addons/account/models/chart_template.py

```diff
@@ -184,7 +184,25 @@ def try_loading(self, company=False):
         if not company.chart_template_id and not self.existing_accounting(company):
             for template in self:
                 template.with_context(default_company_id=company.id)._load(15.0, 15.0, company)
-
+            # Install the demo data when the first localization is instanciated on the company
+            if install_demo and tools.config['demo'].get('account'):
```

Comment:
@william-andre Something is wrong here.  I tested by installing an extra localization through the Apps menu and the tools.config['demo'] did not contain anything.  Would not there be another way to check e.g. if account was installed with demo data?  In any case, the demo data by module is weird anyways.